SPECS/xs-openssl.spec

```diff
@@ -84,13 +84,20 @@ Patch56: openssl-1.1.1-s390x-ecc.patch
 Patch74: openssl-1.1.1-addrconfig.patch
 Patch75: openssl-1.1.1-tls13-curves.patch
 Patch81: openssl-1.1.1-read-buff.patch
+Patch82: openssl-1.1.1-cve-2022-0778.patch
+Patch83: openssl-1.1.1-replace-expired-certs.patch
+Patch84: openssl-1.1.1-cve-2022-1292.patch
+Patch85: openssl-1.1.1-cve-2022-2068.patch
+Patch86: openssl-1.1.1-cve-2022-2097.patch
+#OpenSSL 1.1.1t CVEs
+Patch101: openssl-1.1.1-cve-2022-4304-RSA-oracle.patch
+Patch102: openssl-1.1.1-cve-2022-4450-PEM-bio.patch
+Patch103: openssl-1.1.1-cve-2023-0215-BIO-UAF.patch
+Patch104: openssl-1.1.1-cve-2023-0286-X400.patch
 
 
-Patch100: use-linux-random-header.patch
+Patch500: use-linux-random-header.patch
```

Comment:
Let's document where this patch comes from
```suggestion
# XenServer patch
Patch500: use-linux-random-header.patch
```

Reply:
Done.